README.md

```diff
@@ -69,11 +69,6 @@ The following settings are required to connect to the API.
 ### Remarks
 #### General
  - In the account creation process, the newly created accounts are disabled afterward using an additional web call. This is necessary because the web service does not support creating accounts in a disabled state. However, if an account already exists, the connector does not perform a disable action to avoid unintended behavior.
-- The connector utilizes a custom overview call to retrieve a list of employees. This is necessary because the original web call to retrieve employee data takes 30-60 seconds, which is not feasible in HelloID. The custom overview call includes an ID that may differ between customers, so it is added as a script variable in the Create action.
-    ```Powershell
-      # Script Configuration
```

Comment:
Er staat ook nog een voorbeeld onder het kopje: `## Setup the connector `
deze kan ook weg: `  $employeeOverviewId = '1000000'`

Reply:
Readme aangepast